components/Ui/UiCta.vue

```diff
@@ -20,7 +20,7 @@
       class="cta__icon"
```

Comment:
Class naming should be change on this file too. Example:

```suggestion
      class="ui-cta__icon"
```